source/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,116 @@
+//
+// Footer Component
+//
+// Based on the existing GOV.UK footer in GOV.UK Template, built with flexbox
+// and with tweaked behaviour for the OGL logo for the smaller viewport (when
+// allowing for the table of contents)
+
+@include exports("footer") {
+
+  .app-c-footer {
+    display: block;
+    margin-top: $govuk-spacing-scale-1;
+    padding: $govuk-gutter $govuk-gutter-half ($govuk-gutter * 2);
+    border-top: 1px solid $govuk-footer-border-top;
+    color: $govuk-footer-text;
+    background: $govuk-footer-background;
+    @include mq($from: 768px) {
+      padding: $govuk-gutter;
+    }
+
+    a:link,
+    a:visited {
+      color: $govuk-footer-link;
```

Comment:
I find it a bit weird that these colours are defined so far away from the component, but I think we should address this when we create the footer component in frontend?

Reply:
Yes, feels a bit unnatural to have them anywhere else than next to the component.